v0/src/globalVariables.ts

```
@@ -1,27 +1,46 @@
 /*global ...*/
 /*eslint no-undef: "error"*/
 
-declare const window: any
-
 import jQuery from 'jquery'
+import Array from './simulator/src/arrayHelpers.js'
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)